src/main/java/it/pagopa/pn/deliverypush/middleware/dao/timelinedao/dynamo/mapper/DtoToEntityTimelineMapper.java

```diff
@@ -45,4 +47,8 @@ private LegalFactsIdEntity mapOneLegalFact(LegalFactsIdInt legalFactsId) {
     private TimelineElementDetailsEntity dtoToDetailsEntity(TimelineElementDetailsInt details) {
         return SmartMapper.mapToClass(details, TimelineElementDetailsEntity.class );
     }
+
+    private StatusInfoEntity dtoToStatusInfoEntity(StatusInfoInternal statusInfoInternal) {
+        return SmartMapper.mapToClass(statusInfoInternal, StatusInfoEntity.class );
+    }
```

Comment:
Utilizzerei anche qui il mapping manuale come fatto in entityToDto. Per quanto sia ancora utilizzato in alcuni punti del codice ModelMapper, lo eviterei quando possibile (sono più i problemi che ci ha creato che quelli che ha risolto)

Reply:
fatto nell'ultimo commit.